crates/shared/src/network.rs

```diff
@@ -1,163 +1,19 @@
-        ("322", 322) => "KuCoin",
-        ("32659", 32659) => "Fusion",
-        ("33", 33) => "GoodData",
-        ("35", 35) => "TBWG",
-        ("35855456", 35855456) => "Joys",
-        ("38", 38) => "Valorbit",
-        ("385", 385) => "Lisinski",
-        ("39797", 39797) => "Energi",
-        ("4", 4) => "Ethereum / Rinkeby",
-        ("40", 40) => "Telos",
-        ("41", 41) => "Telos",
-        ("42", 42) => "Ethereum / Kovan",
-        ("420", 420) => "Optimistic",
-        ("42069", 42069) => "pegglecoin",
-        ("42220", 42220) => "Celo",
-        ("43", 43) => "Darwinia",
-        ("43110", 43110) => "Athereum",
-        ("1", 43113) => "Avalanche",
-        ("1", 43114) => "Avalanche",
-        ("44", 44) => "Darwinia",
-        ("44787", 44787) => "Celo",
-        ("4689", 4689) => "IoTeX",
-        ("4690", 4690) => "IoTeX",
-        ("49797", 49797) => "Energi",
-        ("499", 499) => "Rupaya",
         ("5", 5) => "Ethereum / Goerli",
-        ("50", 50) => "XinFin",
-        ("51", 51) => "XinFin",
-        ("52", 52) => "CoinEx",
-        ("53", 53) => "CoinEx",
-        ("558", 558) => "Tao",
-        ("56", 56) => "Binance",
-        ("5869", 5869) => "Wegochain",
-        ("595", 595) => "Acala",
-        ("6", 6) => "Ethereum Classic / Kotti",
-        ("60", 60) => "GoChain",
-        ("1", 61) => "Ethereum Classic / Mainnet",
-        ("61717561", 61717561) => "Aquachain",
-        ("2", 62) => "Ethereum Classic / Morden",
-        ("62320", 62320) => "Celo",
-        ("7", 63) => "Ethereum Classic / Mordor",
-        ("64", 64) => "Ellaism",
-        ("65", 65) => "OKExChain",
-        ("66", 66) => "OKExChain",
-        ("67", 67) => "DBChain",
-        ("68", 68) => "SoterOne",
-        ("686", 686) => "Karura",
-        ("69", 69) => "Optimistic",
-        ("7", 7) => "ThaiChain",
-        ("73799", 73799) => "Energy",
-        ("76", 76) => "Mix",
-        ("77", 77) => "POA",
-        ("7762959", 7762959) => "Musicoin",
-        ("777", 777) => "Ethermint",
-        ("78", 78) => "PrimusChain",
-        ("78110", 78110) => "Firenze",
-        ("787", 787) => "Acala",
-        ("88", 8) => "Ubiq",
-        ("80001", 80001) => "Matic",
-        ("8029", 8029) => "MDGL",
-        ("82", 82) => "Meter",
-        ("1", 820) => "Callisto",
-        ("2", 821) => "Callisto",
-        ("8217", 8217) => "Klaytn",
-        ("8285", 8285) => "KorthoTest",
-        ("88", 88) => "TomoChain",
-        ("888", 888) => "Wanchain",
-        ("8995", 8995) => "bloxberg",
-        ("2", 9) => "Ubiq",
-        ("97", 97) => "Binance",
-        ("977", 977) => "Nepal",
-        ("99", 99) => "POA",
-        ("99415706", 99415706) => "Joys",
-        ("999", 999) => "Wanchain",
-        _ => "<unknown>",
-    }
+        ("100", 100) => "xDAI",
+        ("11155111", 11155111) => "Ethereum / Sepolia",
+        _ => return format!("<unknown network network_id={network_id} chain_id={chain_id}>"),
```

Comment:
Should this be an error?
Not sure if we'd ever want to have this as the network name we use in the backend.
If we don't want to change the signature of the function I think it's reasonable to `panic!()` here and keep returning a `&'static str` from the function.

Reply:
I thought we would be using unknown chains for testing, but this is indeed not the case. I committed this suggestion.